### ISSUE TITLE
flutter run does not work for iOS simulator

### ISSUE THREAD

**Issue by DaveShuckerow** [flutter/flutter#5635]
This is an issue affecting the master build as of 8/26/2016

## Steps to Reproduce

I've tested this out from both Atom and command line.
Start an iOS simulator and nothing else
try to build and deploy leafy app (from Atom, flutter build ios && flutter install, or flutter run)

```Could not find the built application bundle at build/ios/Release-iphonesimulator/Runner.app.```

## Flutter Doctor
[✓] Flutter (on Mac OS, channel master)
    • Flutter at ...
    • Framework revision 4db096ab4e (6 hours ago), 2016-08-26 17:47:12
    • Engine revision c4022b61fa
    • Tools Dart version 1.19.0-dev.5.0

[✓] Android toolchain - develop for Android devices (Android SDK 24.0.0)
    • Android SDK at ...
    • Platform android-N, build-tools 24.0.0
    • Java(TM) SE Runtime Environment (build 1.8.0_91-b14)

[✓] iOS toolchain - develop for iOS devices (Xcode 7.3.1)
    • XCode at /Applications/Xcode.app/Contents/Developer
    • Xcode 7.3.1, Build version 7D1014

[✓] Atom - a lightweight development environment for Flutter
    • flutter plugin version 0.2.4
    • dartlang plugin version 0.6.38

> For more information about diagnosing and reporting Flutter bugs, please see [https://flutter.io/bug-reports/](https://flutter.io/bug-reports/).

**Comment by eseidelGoogle** [comment]
@cbracken recently changed how build directories were configured.  Possibly related?

**Comment by cbracken** [comment]
Taking a look. Question - does uninstalling the app on the simulator, then re-running solve the problem? IIRC there's some caching we do and I wonder if it might trigger this as a one-off.

**Comment by cbracken** [comment]
Quick test on the gallery example and I'm unable to repro, I'll see if I can repro on leafy specifically.

**Comment by DaveShuckerow** [comment]
Our build directory does not appear to match the structure that flutter run
wants.  Running a build for iOS, the directory structure has no matches for
`build/ios/Release-*.

We are stable on alpha channel, so this doesn't need to be fixed
immediately, I'm just looking ahead to unblock future work from API changes.

**Comment by DaveShuckerow** [comment]
Update: Uninstalling does not resolve.  Could it be possible that my version of xcode is creating this issue?

It appears that `flutter build ios` is building to `ios/build/Release-*` while flutter install is looking for `build/ios/Release-*`.

**Comment by cbracken** [comment]
After a bit of poking around the issue was in the leafy Xcode config. Seemed like the Flutter.xcconfig in the project was missing the `#include "Generated.xcconfig"`, which is where the build vars are set. After patching that in, things are looking good again.

It may be worth us adding a sanity check + warning to ensure we find that line in Flutter.xcconfig given how much of the build process depends on it.
